raw_data/preprocessed/africa/egypt_english.txt

```diff
@@ -0,0 +1,243 @@
asts or on surface water for the purpose of conducting commercial, industrial, touristic or scientific activities. [31] War Ship: Any ship that belongs to the armed forces of a state, carrying its distinct external insignia, under the command of an officer officially appointed by the government of such state and operated by a crew governed by the rules of military discipline of that state. [32] Government Ship: A ship owned by the state and operated or used for governmental and non-commercial purposes. [33] Harmful Materials Carrier: A ship constructed originally or redesigned for the purpose of carrying cargoes of harmful liquid substances in bulk, including oil tankers when fully or partially loaded with harmful liquid substances not packed in accordance with the provisions stated in Chapter One, Part Three of this Law. [34] Establishment: Establishment means the following: [•] Industrial establishments subject to the provisions of Law No. 21 of 1958 and Law No. 55 of 1977. [•] Tourist establishments subject to the provisions of Law No. 1 of 1973 and Law No. 1 of 1992. [•] Establishments used for electrical power generation and production which are under the provisions of laws Nos. 145/1948, 63/1974, 12/1976, 13/1976, 27/1976, and 103/1986. [•] Mines, quarries and establishments operating in the field of oil exploration, drilling, transportation and usage which are subject to the provisions of laws Nos. 66/1953, 86/1956, 61/1958 and 4/1988. [•] All infrastructure projects. [•] Any other establishment, activity or project which may have a noticeable impact on the environment . These shall be determined by a decision issued by the Environmental Affairs Agency in agreement with the competent administrative authority. [35] Environmental Monitoring Networks: Agencies which undertake, within their spheres of competence and through their stations and work units, to monitor the components and pollutants of the environment and relay their results and data to the competent authorities periodically. [36] Environmental Impact Assessment: Studying and analyzing the environmental feasibility of proposed projects, whose construction or activities might affect the safety of the environment in order to protect it. [37] Environmental Disaster: An accident resulting from natural factors or human action which causes severe damage to the environment and requires resources beyond the local means. [38] The competent Administrative Agency Concerned with the Protection of the Water Environment: Any of the following agencies, each within its field of competence: [a] The Environmental Affairs Agency (EEAA). [b] The Department of Ports and Lighthouses. [c] The Suez Canal Authority. [d] Port Authorities in ARE. [e] The General Egyptian Organization for the Protection of the Coast. [f] Egyptian General Petroleum Corporation (EGPC). [g] General Department of Surface Water Police. [h] Tourism Development Authority. [i] Other agencies designated by a Prime Ministerial Decree. 
```

Comment:
For the second list following `Any of the following agencies, each within its field of competence:` I have left some of the following first words capitalized because they appear to be definitions or names.

Reply:
I agree with that decision.